x-pack/plugins/uptime/public/components/synthetics/status_badge.tsx

```diff
@@ -45,9 +46,18 @@ export function textFromStatus(status?: string) {
   }
 }
 
-export const StatusBadge: FC<StatusBadgeProps> = ({ status }) => {
+export const StatusBadge: FC<StatusBadgeProps> = ({ status, stepNo }) => {
   const theme = useContext(UptimeThemeContext);
   return (
-    <EuiBadge color={colorFromStatus(theme.colors, status)}>{textFromStatus(status)}</EuiBadge>
+    <EuiFlexGroup alignItems="center" gutterSize="s">
+      <EuiFlexItem grow={false}>
+        <EuiText>
+          <strong>{stepNo}.</strong>
```

Comment:
Is this considered content of strong importance? Not entirely sure the context in which it appears, but I wouldn't use strong for bold styling unless it's content of extreme importance.

Reply:
i think it's the same struggle as above, EuiText doesn't provide a way to add font-weight without custom styling.